uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/API/Wrapper/LeaveRequestDatesTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+use CRM_HRLeaveAndAbsences_API_Wrapper_LeaveRequestDates as LeaveRequestDates;
+
+/**
+ * Class CRM_HRLeaveAndAbsences_API_Wrapper_LeaveRequestDates
+ *
+ * @group headless
+ */
+class CRM_HRLeaveAndAbsences_API_Wrapper_LeaveRequestDatesTest extends BaseHeadlessTest {
+
+  /**
+   * @var \CRM_HRLeaveAndAbsences_API_Wrapper_LeaveRequestDates
+   */
+  private $wrapper;
+
+  public function setUp() {
```

Comment:
Just want to check, when we say description for all methods, I guess tests are excluded

Reply:
Yeah, they are